packages/console-client/src/client/http/axios/axiosRequestAdapter.ts

```diff
@@ -0,0 +1,169 @@
+    }
+    if (!parseNodeFactory) {
+      throw new Error('parse node factory cannot be null')
+    }
+    if (!serializationWriterFactory) {
+      throw new Error('serialization writer factory cannot be null')
+    }
+    if (!httpClient) {
+      throw new Error('http client cannot be null')
+    }
+  }
+
+  public getSerializationWriterFactory(): SerializationWriterFactory {
+    return this.serializationWriterFactory
+  }
+
+  public send<ModelType extends Parsable>(
+    requestInfo: RequestInformation,
+    deserialization: ParsableFactory<ModelType>,
+    errorMappings: ErrorMappings | undefined
+  ): Promise<ModelType | undefined> {
+    throw new Error('Method not implemented.')
+  }
+
+  public async sendCollection<ModelType extends Parsable>(
+    requestInfo: RequestInformation,
+    deserialization: ParsableFactory<ModelType>,
+    errorMappings: ErrorMappings | undefined
+  ): Promise<ModelType[] | undefined> {
+    const response = await this.getHttpResponseMessage<ModelType[]>(requestInfo)
+    return response.data
+  }
+
+  public sendCollectionOfPrimitive<ResponseType extends Exclude<PrimitiveTypesForDeserializationType, ArrayBuffer>>(
+    requestInfo: RequestInformation,
+    responseType: Exclude<PrimitiveTypesForDeserialization, 'ArrayBuffer'>,
+    errorMappings: ErrorMappings | undefined
+  ): Promise<ResponseType[] | undefined> {
+    throw new Error('Method not implemented.')
+  }
+
+  public sendPrimitive<ResponseType extends PrimitiveTypesForDeserializationType>(
+    requestInfo: RequestInformation,
+    responseType: PrimitiveTypesForDeserialization,
+    errorMappings: ErrorMappings | undefined
+  ): Promise<ResponseType | undefined> {
+    throw new Error('Method not implemented.')
+  }
+
+  public sendNoResponseContent(
+    requestInfo: RequestInformation,
+    errorMappings: ErrorMappings | undefined
+  ): Promise<void> {
+    throw new Error('Method not implemented.')
+  }
+
+  public sendEnum<EnumObject extends Record<string, unknown>>(
+    requestInfo: RequestInformation,
+    enumObject: EnumObject,
+    errorMappings: ErrorMappings | undefined
+  ): Promise<EnumObject[keyof EnumObject] | undefined> {
+    throw new Error('Method not implemented.')
+  }
+
+  public sendCollectionOfEnum<EnumObject extends Record<string, unknown>>(
+    requestInfo: RequestInformation,
+    enumObject: EnumObject,
+    errorMappings: ErrorMappings | undefined
+  ): Promise<EnumObject[keyof EnumObject][] | undefined> {
+    throw new Error('Method not implemented.')
+  }
+
+  public enableBackingStore(
+    backingStoreFactory?: BackingStoreFactory | undefined
+  ): void {
+    throw new Error('Method not implemented.')
+  }
+
+  public convertToNativeRequest<T>(
+    requestInfo: RequestInformation): Promise<T> {
+    throw new Error('Method not implemented.')
+  }
+
+  private async getHttpResponseMessage<T>(requestInfo: RequestInformation, claims?: string): Promise<AxiosClientResponse<T>> {
```

Comment:
what is the purpose of the claims string? let's remove it if useless

Reply:
removed